minutes/2019-11-12.md

```diff
@@ -0,0 +1,65 @@
+# Jakarta EE Platform Call #2
+
+General
+Date: 2019-11-12
+
+Present:
+
+* Kevin Sutter
+* Bill Shannon
+* Ivar Grimstad
+* Emily Jiang
+* Tanja Obradovic
+* Martin Stefanko
+* Ken Finnigan
+* Edwin Amoakwa
+* Carlos Andres De La Rosa
+* Steve Millidge
+* Rudy De Busscher
+* Alessio Soldano
+
+## Agenda
+
+* Organize ourselves
+
+Kevin (IBM) and Steve (Payara) volunteered to take on the Delivery Owner and Co-pilot role for Jakarta EE 9. Final confirmation pending.
+
+Keep the current meeting time: 11:00 AM EST.
+
+Minutes will be published at 
+(Jakarta EE Platform Project Web Page)https://eclipse-ee4j.github.io/jakartaee-platform/ 
+
+Source of minutes:
+https://github.com/eclipse-ee4j/jakartaee-platform/blob/gh-pages/minutes
+
+
+* [KWS] Big Bang vs Incremental.  Need to decide.  Several projects are stomping at the bit to move on the package rename exercise.
+
+**DECISION**:  Big Bang is the stated direction.
+
+* [KWS] Java SE 8 or Java SE 11 as the minimum.  Another key decision to allow for continued development efforts.
+
+* [IG] “Jakartify” the specification documents.
+Status: https://github.com/eclipse-ee4j/jakartaee-platform/projects/2
+
+* [IG] Decide which specifications to prune/deprecate.
+Publish specific proposal(s) to Platform dev mailing list.  Steve volunteered to organize the material and post for a “vote” by Platform Committers.
+
+* [TO] Tanja to find out a list of committers on the Jakarta EE Platform Project and invite them to sign up to the platform project mailing list - look here
+
+* [KWS] Determine a dependency graph for the remaining projects and work out a spreadsheet (or some doc) of all of the pieces of required work.
+   * Work with David Blevins on the dependency work that he did previously.
+
+
+Steering Committee, October 29, 2019:
+
```

Comment:
May need an intro sentence into what the below resolution means to the project

Reply:
Agree, @smillidge.  How about something like this?

The following resolution from the Jakarta EE Steering Committee was delivered to this Platform Project.  It is now our project's duty to analyze the request and determine validity of the request, along with developing a project plan to support the request by Dec 9.  This effort will require (intense) discussions on the mailing list as well as our weekly phone calls to help resolve issues and determine a delivery plan acceptable to both parties.

Reply:
I think the request form the SC was stronger than that, and while I hesitate to use the word "constraint", I think it's pretty close to that.

How about...

Based on previous discussions, the Jakarta EE Steering Committee provided the following guidance in the form of goals for the Jakarta EE 9 release.  This project team needs to produce a plan that meets these goals by Dec 9.  If we determine that there are technical reasons that a particular goal can not be met, we need to provide that feedback to the Steering Committee.  The project plan needs to provide technical guidance to the many specification and implementation projects that will contribute to the Jakarta EE 9 release, and must determine whether sufficient resources are likely to be available to meet the Jakarta EE9 requirements on the proposed schedule.

Reply:
I have added the text above to the PR